### DISCUSSION TITLE
SQL Server Schema.Table("tablename").Column("ColumnName").Exists()) for schema

### DISCUSSION THREAD

**Discussion by turagittech** [fluentmigrator/fluentmigrator#1826] [Q&A]
Hi , 
I was wanting to use the above construct to verify a column exists on a table where the schema is not dbo, but this doesn't appear to have a suitable approach. IS there some way I can set the schema before calling this? 
I've seen things about calling the runner with a different schema, but that was not what I wanted as I already have various objects built in different schemas. If there is a feature request please advise
example of code

if (Schema.Table("TableName").Column("ColumnName").Exists())
            {
               Alter.Table("TableName")
                    .InSchema("dv_raw")
                    .AddColumn("ColumnName").AsString(128).Nullable();
            }

**Comment by PhenX** [comment]
This should do the trick : 

```c#
if (Schema.Schema("dv_raw").Table("TableName").Column("ColumnName").Exists()) // << here .Schema("your_schema")
{
  Alter.Table("TableName")
    .InSchema("dv_raw")
    .AddColumn("ColumnName").AsString(128).Nullable();
}
```
